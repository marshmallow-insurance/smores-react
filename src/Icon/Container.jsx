import React from 'react'
import { Icon } from './Icon'
import styled from 'styled-components'

export const Container = () => {
  return (
    <div>
      <Wrapper>
        <span>address-mta</span>
        <Icon render="address-mta" />
      </Wrapper>
      <Wrapper>
        <span>alert-engine</span>
        <Icon render="alert-engine" />
      </Wrapper>
      <Wrapper>
        <span>arrow</span>
        <Icon render="arrow" />
      </Wrapper>
      <Wrapper>
        <span>arrow-curved</span>
        <Icon render="arrow-curved" />
      </Wrapper>
      <Wrapper>
        <span>at</span>
        <Icon render="at" />
      </Wrapper>
      <Wrapper>
        <span>breakdown</span>
        <Icon render="breakdown" />
      </Wrapper>
      <Wrapper>
        <span>briefcase</span>
        <Icon render="briefcase" />
      </Wrapper>
      <Wrapper>
        <span>burger</span>
        <Icon render="burger" />
      </Wrapper>
      <Wrapper>
        <span>calendar</span>
        <Icon render="calendar" />
      </Wrapper>
      <Wrapper>
        <span>camera</span>
        <Icon render="camera" />
      </Wrapper>
      <Wrapper>
        <span>car</span>
        <Icon render="car" />
      </Wrapper>
      <Wrapper>
        <span>car-claim</span>
        <Icon render="car-claim" />
      </Wrapper>
      <Wrapper>
        <span>car-mta</span>
        <Icon render="car-mta" />
      </Wrapper>
      <Wrapper>
        <span>card</span>
        <Icon render="card" />
      </Wrapper>
      <Wrapper>
        <span>caret</span>
        <Icon render="caret" />
      </Wrapper>
      <Wrapper>
        <span>charge</span>
        <Icon render="charge" />
      </Wrapper>
      <Wrapper>
        <span>chat</span>
        <Icon render="chat" />
      </Wrapper>
      <Wrapper>
        <span>chat-bubble</span>
        <Icon render="chat-bubble" />
      </Wrapper>
      <Wrapper>
        <span>city</span>
        <Icon render="city" />
      </Wrapper>
      <Wrapper>
        <span>clock</span>
        <Icon render="clock" />
      </Wrapper>
      <Wrapper>
        <span>contact-email</span>
        <Icon render="contact-email" />
      </Wrapper>
      <Wrapper>
        <span>cross</span>
        <Icon render="cross" />
      </Wrapper>
      <Wrapper>
        <span>doc</span>
        <Icon render="doc" />
      </Wrapper>
      <Wrapper>
        <span>doc-download</span>
        <Icon render="doc-download" />
      </Wrapper>
      <Wrapper>
        <span>download</span>
        <Icon render="download" />
      </Wrapper>
      <Wrapper>
        <span>edit</span>
        <Icon render="edit" />
      </Wrapper>
      <Wrapper>
        <span>europe</span>
        <Icon render="europe" />
      </Wrapper>
      <Wrapper>
        <span>file</span>
        <Icon render="file" />
      </Wrapper>
      <Wrapper>
        <span>fire</span>
        <Icon render="fire" />
      </Wrapper>
      <Wrapper>
        <span>house</span>
        <Icon render="house" />
      </Wrapper>
      <Wrapper>
        <span>id</span>
        <Icon render="id" />
      </Wrapper>
      <Wrapper>
        <span>info</span>
        <Icon render="info" />
      </Wrapper>
      <Wrapper>
        <span>injury</span>
        <Icon render="injury" />
      </Wrapper>
      <Wrapper>
        <span>intercom</span>
        <Icon render="intercom" />
      </Wrapper>
      <Wrapper>
        <span>key</span>
        <Icon render="key" />
      </Wrapper>
      <Wrapper>
        <span>minus</span>
        <Icon render="minus" />
      </Wrapper>
      <Wrapper>
        <span>money-bag</span>
        <Icon render="money-bag" />
      </Wrapper>
      <Wrapper>
        <span>new-window</span>
        <Icon render="new-window" />
      </Wrapper>
      <Wrapper>
        <span>padlock</span>
        <Icon render="padlock" />
      </Wrapper>
      <Wrapper>
        <span>people</span>
        <Icon render="people" />
      </Wrapper>
      <Wrapper>
        <span>person</span>
        <Icon render="person" />
      </Wrapper>
      <Wrapper>
        <span>plus</span>
        <Icon render="plus" />
      </Wrapper>
      <Wrapper>
        <span>plus-circle</span>
        <Icon render="plus-circle" />
      </Wrapper>
      <Wrapper>
        <span>policy</span>
        <Icon render="policy" />
      </Wrapper>
      <Wrapper>
        <span>pound-filled-circle</span>
        <Icon render="pound-filled-circle" />
      </Wrapper>
      <Wrapper>
        <span>pound-outline</span>
        <Icon render="pound-outline" />
      </Wrapper>
      <Wrapper>
        <span>profile</span>
        <Icon render="profile" />
      </Wrapper>
      <Wrapper>
        <span>question</span>
        <Icon render="question" />
      </Wrapper>
      <Wrapper>
        <span>refund</span>
        <Icon render="refund" />
      </Wrapper>
      <Wrapper>
        <span>renewal</span>
        <Icon render="renewal" />
      </Wrapper>
      <Wrapper>
        <span>schedule-quote</span>
        <Icon render="schedule-quote" />
      </Wrapper>
      <Wrapper>
        <span>search</span>
        <Icon render="search" />
      </Wrapper>
      <Wrapper>
        <span>send-message</span>
        <Icon render="send-message" />
      </Wrapper>
      <Wrapper>
        <span>shield</span>
        <Icon render="shield" />
      </Wrapper>
      <Wrapper>
        <span>tick</span>
        <Icon render="tick" />
      </Wrapper>
      <Wrapper>
        <span>tow-truck</span>
        <Icon render="tow-truck" />
      </Wrapper>
      <Wrapper>
        <span>trash</span>
        <Icon render="trash" />
      </Wrapper>
      <Wrapper>
        <span>up-down</span>
        <Icon render="up-down" />
      </Wrapper>
      <Wrapper>
        <span>upload-email</span>
        <Icon render="upload-email" />
      </Wrapper>
      <Wrapper>
        <span>village</span>
        <Icon render="village" />
      </Wrapper>
      <Wrapper>
        <span>warning</span>
        <Icon render="warning" />
      </Wrapper>
      <Wrapper>
        <span>wheel-flat</span>
        <Icon render="wheel-flat" />
      </Wrapper>
      <Wrapper>
        <span>windscreen</span>
        <Icon render="windscreen" />
      </Wrapper>
      <Wrapper>
        <span>windscreen-repair</span>
        <Icon render="windscreen-repair" />
      </Wrapper>
      <Wrapper>
        <span>windscreen-replacement</span>
        <Icon render="windscreen-replacement" />
      </Wrapper>
    </div>
  )
}

const Wrapper = styled.div`
  align-items: center;
  display: flex;
  justify-content: space-between;
  margin-bottom: 10px;
  width: 250px;
`
