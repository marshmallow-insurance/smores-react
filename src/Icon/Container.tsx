import React from 'react'
import { Icon } from './Icon'
import styled from 'styled-components'

export const Container = () => {
  return (
    <div>
      <Wrapper>
        <span>aa</span>
        <Icon render="aa" />
      </Wrapper>
      <Wrapper>
        <span>aa-inverted</span>
        <Icon render="aa-inverted" />
      </Wrapper>
      <Wrapper>
        <span>accidental-damage</span>
        <Icon render="accidental-damage" />
      </Wrapper>
      <Wrapper>
        <span>account-breakdown</span>
        <Icon render="account-breakdown" />
      </Wrapper>
      <Wrapper>
        <span>account-mlp</span>
        <Icon render="account-mlp" />
      </Wrapper>
      <Wrapper>
        <span>activity-clipboard</span>
        <Icon render="activity-clipboard" />
      </Wrapper>
      <Wrapper>
        <span>add-ons</span>
        <Icon render="add-ons" />
      </Wrapper>
      <Wrapper>
        <span>add-ons</span>
        <Icon render="add-ons" />
      </Wrapper>
      <Wrapper>
        <span>addons</span>
        <Icon render="addons" />
      </Wrapper>
      <Wrapper>
        <span>address-mta</span>
        <Icon render="address-mta" />
      </Wrapper>
      <Wrapper>
        <span>album-audio</span>
        <Icon render="album-audio" />
      </Wrapper>
      <Wrapper>
        <span>alert-engine</span>
        <Icon render="alert-engine" />
      </Wrapper>
      <Wrapper>
        <span>alert</span>
        <Icon render="alert" />
      </Wrapper>
      <Wrapper>
        <span>amex</span>
        <Icon render="amex" />
      </Wrapper>
      <Wrapper>
        <span>apple-pay</span>
        <Icon render="apple-pay" />
      </Wrapper>
      <Wrapper>
        <span>approved-file</span>
        <Icon render="approved-file" />
      </Wrapper>
      <Wrapper>
        <span>arrow</span>
        <Icon render="arrow" />
      </Wrapper>
      <Wrapper>
        <span>at</span>
        <Icon render="at" />
      </Wrapper>
      <Wrapper>
        <span>axa</span>
        <Icon render="axa" />
      </Wrapper>
      <Wrapper>
        <span>bank</span>
        <Icon render="bank" />
      </Wrapper>
      <Wrapper>
        <span>basket</span>
        <Icon render="basket" />
      </Wrapper>
      <Wrapper>
        <span>bicycle</span>
        <Icon render="bicycle" />
      </Wrapper>
      <Wrapper>
        <span>boost</span>
        <Icon render="boost" />
      </Wrapper>
      <Wrapper>
        <span>breakdown-alt</span>
        <Icon render="breakdown-alt" />
      </Wrapper>
      <Wrapper>
        <span>breakdown</span>
        <Icon render="breakdown" />
      </Wrapper>
      <Wrapper>
        <span>briefcase</span>
        <Icon render="briefcase" />
      </Wrapper>
      <Wrapper>
        <span>burger</span>
        <Icon render="burger" />
      </Wrapper>
      <Wrapper>
        <span>calendar</span>
        <Icon render="calendar" />
      </Wrapper>
      <Wrapper>
        <span>camera</span>
        <Icon render="camera" />
      </Wrapper>
      <Wrapper>
        <span>cancel</span>
        <Icon render="cancel" />
      </Wrapper>
      <Wrapper>
        <span>cancel-bw</span>
        <Icon render="cancel-bw" />
      </Wrapper>
      <Wrapper>
        <span>car-mta</span>
        <Icon render="car-mta" />
      </Wrapper>
      <Wrapper>
        <span>car-usage</span>
        <Icon render="car-usage" />
      </Wrapper>
      <Wrapper>
        <span>car</span>
        <Icon render="car" />
      </Wrapper>
      <Wrapper>
        <span>card</span>
        <Icon render="card" />
      </Wrapper>
      <Wrapper>
        <span>caret</span>
        <Icon render="caret" />
      </Wrapper>
      <Wrapper>
        <span>charge</span>
        <Icon render="charge" />
      </Wrapper>
      <Wrapper>
        <span>chat-bubble</span>
        <Icon render="chat-bubble" />
      </Wrapper>
      <Wrapper>
        <span>chat</span>
        <Icon render="chat" />
      </Wrapper>
      <Wrapper>
        <span>checkout</span>
        <Icon render="checkout" />
      </Wrapper>
      <Wrapper>
        <span>checkout-with-text</span>
        <Icon render="checkout-with-text" />
      </Wrapper>
      <Wrapper>
        <span>circle-tick</span>
        <Icon render="circle-tick" />
      </Wrapper>
      <Wrapper>
        <span>city</span>
        <Icon render="city" />
      </Wrapper>
      <Wrapper>
        <span>claim</span>
        <Icon render="claim" />
      </Wrapper>
      <Wrapper>
        <span>claim-line</span>
        <Icon render="claim-line" />
      </Wrapper>
      <Wrapper>
        <span>clipboard-check</span>
        <Icon render="clipboard-check" />
      </Wrapper>
      <Wrapper>
        <span>clock</span>
        <Icon render="clock" />
      </Wrapper>
      <Wrapper>
        <span>close-panel</span>
        <Icon render="close-panel" />
      </Wrapper>
      <Wrapper>
        <span>confused</span>
        <Icon render="confused" />
      </Wrapper>
      <Wrapper>
        <span>confused-blue</span>
        <Icon render="confused-blue" />
      </Wrapper>
      <Wrapper>
        <span>contact-email</span>
        <Icon render="contact-email" />
      </Wrapper>
      <Wrapper>
        <span>copy</span>
        <Icon render="copy" />
      </Wrapper>
      <Wrapper>
        <span>crop</span>
        <Icon render="crop" />
      </Wrapper>
      <Wrapper>
        <span>cross</span>
        <Icon render="cross" />
      </Wrapper>
      <Wrapper>
        <span>dashboard-high</span>
        <Icon render="dashboard-high" />
      </Wrapper>
      <Wrapper>
        <span>direct-debit</span>
        <Icon render="direct-debit" />
      </Wrapper>
      <Wrapper>
        <span>doc-download</span>
        <Icon render="doc-download" />
      </Wrapper>
      <Wrapper>
        <span>doc-upload</span>
        <Icon render="doc-upload" />
      </Wrapper>
      <Wrapper>
        <span>download</span>
        <Icon render="download" />
      </Wrapper>
      <Wrapper>
        <span>earth</span>
        <Icon render="earth" />
      </Wrapper>
      <Wrapper>
        <span>edit</span>
        <Icon render="edit" />
      </Wrapper>
      <Wrapper>
        <span>edit-contact</span>
        <Icon render="edit-contact" />
      </Wrapper>
      <Wrapper>
        <span>edit-outline</span>
        <Icon render="edit-outline" />
      </Wrapper>
      <Wrapper>
        <span>email-filled</span>
        <Icon render="email-filled" />
      </Wrapper>
      <Wrapper>
        <span>engine</span>
        <Icon render="engine" />
      </Wrapper>
      <Wrapper>
        <span>europe</span>
        <Icon render="europe" />
      </Wrapper>
      <Wrapper>
        <span>excluded</span>
        <Icon render="excluded" />
      </Wrapper>
      <Wrapper>
        <span>expand</span>
        <Icon render="expand" />
      </Wrapper>
      <Wrapper>
        <span>facebook</span>
        <Icon render="facebook" />
      </Wrapper>
      <Wrapper>
        <span>falling-items</span>
        <Icon render="falling-items" />
      </Wrapper>
      <Wrapper>
        <span>fence</span>
        <Icon render="fence" />
      </Wrapper>
      <Wrapper>
        <span>file-badge-2</span>
        <Icon render="file-badge-2" />
      </Wrapper>
      <Wrapper>
        <span>file-cabinet</span>
        <Icon render="file-cabinet" />
      </Wrapper>
      <Wrapper>
        <span>file</span>
        <Icon render="file" />
      </Wrapper>
      <Wrapper>
        <span>filter</span>
        <Icon render="filter" />
      </Wrapper>
      <Wrapper>
        <span>fire</span>
        <Icon render="fire" />
      </Wrapper>
      <Wrapper>
        <span>flag</span>
        <Icon render="flag" />
      </Wrapper>
      <Wrapper>
        <span>folder</span>
        <Icon render="folder" />
      </Wrapper>
      <Wrapper>
        <span>fraud-alert</span>
        <Icon render="fraud-alert" />
      </Wrapper>
      <Wrapper>
        <span>freezer-food</span>
        <Icon render="freezer-food" />
      </Wrapper>
      <Wrapper>
        <span>fuel</span>
        <Icon render="fuel" />
      </Wrapper>
      <Wrapper>
        <span>globe</span>
        <Icon render="globe" />
      </Wrapper>
      <Wrapper>
        <span>google-play-badge</span>
        <Icon render="google-play-badge" />
      </Wrapper>
      <Wrapper>
        <span>gpay</span>
        <Icon render="gpay" />
      </Wrapper>
      <Wrapper>
        <span>help-aboutus</span>
        <Icon render="help-aboutus" />
      </Wrapper>
      <Wrapper>
        <span>help-buypolicy</span>
        <Icon render="help-buypolicy" />
      </Wrapper>
      <Wrapper>
        <span>help-center</span>
        <Icon render="help-center" />
      </Wrapper>
      <Wrapper>
        <span>help-idcheck</span>
        <Icon render="help-idcheck" />
      </Wrapper>
      <Wrapper>
        <span>help-manage-policy</span>
        <Icon render="help-manage-policy" />
      </Wrapper>
      <Wrapper>
        <span>help-ncd</span>
        <Icon render="help-ncd" />
      </Wrapper>
      <Wrapper>
        <span>help-verification</span>
        <Icon render="help-verification" />
      </Wrapper>
      <Wrapper>
        <span>home-excluded</span>
        <Icon render="home-excluded" />
      </Wrapper>
      <Wrapper>
        <span>house-tree</span>
        <Icon render="house-tree" />
      </Wrapper>
      <Wrapper>
        <span>house</span>
        <Icon render="house" />
      </Wrapper>
      <Wrapper>
        <span>included</span>
        <Icon render="included" />
      </Wrapper>
      <Wrapper>
        <span>id</span>
        <Icon render="id" />
      </Wrapper>
      <Wrapper>
        <span>info</span>
        <Icon render="info" />
      </Wrapper>
      <Wrapper>
        <span>injury</span>
        <Icon render="injury" />
      </Wrapper>
      <Wrapper>
        <span>instagram</span>
        <Icon render="instagram" />
      </Wrapper>
      <Wrapper>
        <span>intercom</span>
        <Icon render="intercom" />
      </Wrapper>
      <Wrapper>
        <span>intercom2</span>
        <Icon render="intercom2" />
      </Wrapper>
      <Wrapper>
        <span>ios-store-badge</span>
        <Icon render="ios-store-badge" />
      </Wrapper>
      <Wrapper>
        <span>iphone</span>
        <Icon render="iphone" />
      </Wrapper>
      <Wrapper>
        <span>jewellery</span>
        <Icon render="jewellery" />
      </Wrapper>
      <Wrapper>
        <span>key</span>
        <Icon render="key" />
      </Wrapper>
      <Wrapper>
        <span>laptop</span>
        <Icon render="laptop" />
      </Wrapper>
      <Wrapper>
        <span>leaks</span>
        <Icon render="leaks" />
      </Wrapper>
      <Wrapper>
        <span>lightbulb</span>
        <Icon render="lightbulb" />
      </Wrapper>
      <Wrapper>
        <span>link</span>
        <Icon render="link" />
      </Wrapper>
      <Wrapper>
        <span>linkedin</span>
        <Icon render="linkedin" />
      </Wrapper>
      <Wrapper>
        <span>live-chat</span>
        <Icon render="live-chat" />
      </Wrapper>
      <Wrapper>
        <span>live-chat2</span>
        <Icon render="live-chat2" />
      </Wrapper>
      <Wrapper>
        <span>location</span>
        <Icon render="location" />
      </Wrapper>
      <Wrapper>
        <span>log-out</span>
        <Icon render="log-out" />
      </Wrapper>
      <Wrapper>
        <span>manage-drivers</span>
        <Icon render="manage-drivers" />
      </Wrapper>
      <Wrapper>
        <span>manage-policy</span>
        <Icon render="manage-policy" />
      </Wrapper>
      <Wrapper>
        <span>marshmallow</span>
        <Icon render="marshmallow" />
      </Wrapper>
      <Wrapper>
        <span>mastercard</span>
        <Icon render="mastercard" />
      </Wrapper>
      <Wrapper>
        <span>maximise</span>
        <Icon render="maximise" />
      </Wrapper>
      <Wrapper>
        <span>medical</span>
        <Icon render="medical" />
      </Wrapper>
      <Wrapper>
        <span>minimise</span>
        <Icon render="minimise" />
      </Wrapper>
      <Wrapper>
        <span>minus-circle</span>
        <Icon render="minus-circle" />
      </Wrapper>
      <Wrapper>
        <span>minus</span>
        <Icon render="minus" />
      </Wrapper>
      <Wrapper>
        <span>money-bag</span>
        <Icon render="money-bag" />
      </Wrapper>
      <Wrapper>
        <span>money-heart</span>
        <Icon render="money-heart" />
      </Wrapper>
      <Wrapper>
        <span>more-dots</span>
        <Icon render="more-dots" />
      </Wrapper>
      <Wrapper>
        <span>mulsanne</span>
        <Icon render="mulsanne" />
      </Wrapper>
      <Wrapper>
        <span>new-window</span>
        <Icon render="new-window" />
      </Wrapper>
      <Wrapper>
        <span>no-excess</span>
        <Icon render="no-excess" />
      </Wrapper>
      <Wrapper>
        <span>notes</span>
        <Icon render="notes" />
      </Wrapper>
      <Wrapper>
        <span>notification</span>
        <Icon render="notification" />
      </Wrapper>
      <Wrapper>
        <span>onfido</span>
        <Icon render="onfido" />
      </Wrapper>
      <Wrapper>
        <span>open-panel</span>
        <Icon render="open-panel" />
      </Wrapper>
      <Wrapper>
        <span>other</span>
        <Icon render="other" />
      </Wrapper>
      <Wrapper>
        <span>out-of-home</span>
        <Icon render="out-of-home" />
      </Wrapper>
      <Wrapper>
        <span>padlock-outline</span>
        <Icon render="padlock-outline" />
      </Wrapper>
      <Wrapper>
        <span>padlock</span>
        <Icon render="padlock" />
      </Wrapper>
      <Wrapper>
        <span>people</span>
        <Icon render="people" />
      </Wrapper>
      <Wrapper>
        <span>person</span>
        <Icon render="person" />
      </Wrapper>
      <Wrapper>
        <span>phone-filled</span>
        <Icon render="phone-filled" />
      </Wrapper>
      <Wrapper>
        <span>phone-outline</span>
        <Icon render="phone-outline" />
      </Wrapper>
      <Wrapper>
        <span>photo</span>
        <Icon render="photo" />
      </Wrapper>
      <Wrapper>
        <span>pin-code</span>
        <Icon render="pin-code" />
      </Wrapper>
      <Wrapper>
        <span>plus-circle</span>
        <Icon render="plus-circle" />
      </Wrapper>
      <Wrapper>
        <span>plus</span>
        <Icon render="plus" />
      </Wrapper>
      <Wrapper>
        <span>policy-details</span>
        <Icon render="policy-details" />
      </Wrapper>
      <Wrapper>
        <span>policy-doc</span>
        <Icon render="policy-doc" />
      </Wrapper>
      <Wrapper>
        <span>policy-docs</span>
        <Icon render="policy-docs" />
      </Wrapper>
      <Wrapper>
        <span>policy</span>
        <Icon render="policy" />
      </Wrapper>
      <Wrapper>
        <span>pound-bold</span>
        <Icon render="pound-bold" />
      </Wrapper>
      <Wrapper>
        <span>pound-filled-circle</span>
        <Icon render="pound-filled-circle" />
      </Wrapper>
      <Wrapper>
        <span>pound-medium</span>
        <Icon render="pound-medium" />
      </Wrapper>
      <Wrapper>
        <span>pound-outline</span>
        <Icon render="pound-outline" />
      </Wrapper>
      <Wrapper>
        <span>pound-regular</span>
        <Icon render="pound-regular" />
      </Wrapper>
      <Wrapper>
        <span>premfina</span>
        <Icon render="premfina" />
      </Wrapper>
      <Wrapper>
        <span>profile</span>
        <Icon render="profile" />
      </Wrapper>
      <Wrapper>
        <span>prohibited</span>
        <Icon render="prohibited" />
      </Wrapper>
      <Wrapper>
        <span>property</span>
        <Icon render="property" />
      </Wrapper>
      <Wrapper>
        <span>question-help</span>
        <Icon render="question-help" />
      </Wrapper>
      <Wrapper>
        <span>rac</span>
        <Icon render="rac" />
      </Wrapper>
      <Wrapper>
        <span>ravelin</span>
        <Icon render="ravelin" />
      </Wrapper>
      <Wrapper>
        <span>refer-a-friend</span>
        <Icon render="refer-a-friend" />
      </Wrapper>
      <Wrapper>
        <span>refund</span>
        <Icon render="refund" />
      </Wrapper>
      <Wrapper>
        <span>reg-plate</span>
        <Icon render="reg-plate" />
      </Wrapper>
      <Wrapper>
        <span>renewal</span>
        <Icon render="renewal" />
      </Wrapper>
      <Wrapper>
        <span>renewal-line</span>
        <Icon render="renewal-line" />
      </Wrapper>
      <Wrapper>
        <span>repeat-charge</span>
        <Icon render="repeat-charge" />
      </Wrapper>
      <Wrapper>
        <span>return</span>
        <Icon render="return" />
      </Wrapper>
      <Wrapper>
        <span>schedule-quote</span>
        <Icon render="schedule-quote" />
      </Wrapper>
      <Wrapper>
        <span>search</span>
        <Icon render="search" />
      </Wrapper>
      <Wrapper>
        <span>security</span>
        <Icon render="security" />
      </Wrapper>
      <Wrapper>
        <span>send-message</span>
        <Icon render="send-message" />
      </Wrapper>
      <Wrapper>
        <span>settings</span>
        <Icon render="settings" />
      </Wrapper>
      <Wrapper>
        <span>shield-check</span>
        <Icon render="shield-check" />
      </Wrapper>
      <Wrapper>
        <span>shield</span>
        <Icon render="shield" />
      </Wrapper>
      <Wrapper>
        <span>shopping</span>
        <Icon render="shopping" />
      </Wrapper>
      <Wrapper>
        <span>sofa</span>
        <Icon render="sofa" />
      </Wrapper>
      <Wrapper>
        <span>stack</span>
        <Icon render="stack" />
      </Wrapper>
      <Wrapper>
        <span>stripe</span>
        <Icon render="stripe" />
      </Wrapper>
      <Wrapper>
        <span>storm</span>
        <Icon render="storm" />
      </Wrapper>
      <Wrapper>
        <span>subsidence</span>
        <Icon render="subsidence" />
      </Wrapper>
      <Wrapper>
        <span>theft</span>
        <Icon render="theft" />
      </Wrapper>
      <Wrapper>
        <span>thumbs-up</span>
        <Icon render="thumbs-up" />
      </Wrapper>
      <Wrapper>
        <span>tick</span>
        <Icon render="tick" />
      </Wrapper>
      <Wrapper>
        <span>tow-truck</span>
        <Icon render="tow-truck" />
      </Wrapper>
      <Wrapper>
        <span>trace-and-access</span>
        <Icon render="trace-and-access" />
      </Wrapper>
      <Wrapper>
        <span>training</span>
        <Icon render="training" />
      </Wrapper>
      <Wrapper>
        <span>trash</span>
        <Icon render="trash" />
      </Wrapper>
      <Wrapper>
        <span>twitter</span>
        <Icon render="twitter" />
      </Wrapper>
      <Wrapper>
        <span>up-down</span>
        <Icon render="up-down" />
      </Wrapper>
      <Wrapper>
        <span>upload-documents</span>
        <Icon render="upload-documents" />
      </Wrapper>
      <Wrapper>
        <span>upload-email</span>
        <Icon render="upload-email" />
      </Wrapper>
      <Wrapper>
        <span>upload</span>
        <Icon render="upload" />
      </Wrapper>
      <Wrapper>
        <span>vandalism</span>
        <Icon render="vandalism" />
      </Wrapper>
      <Wrapper>
        <span>village</span>
        <Icon render="village" />
      </Wrapper>
      <Wrapper>
        <span>visa</span>
        <Icon render="visa" />
      </Wrapper>
      <Wrapper>
        <span>warning</span>
        <Icon render="warning" />
      </Wrapper>
      <Wrapper>
        <span>watch</span>
        <Icon render="watch" />
      </Wrapper>
      <Wrapper>
        <span>wave</span>
        <Icon render="wave" />
      </Wrapper>
      <Wrapper>
        <span>wellbeing</span>
        <Icon render="wellbeing" />
      </Wrapper>
      <Wrapper>
        <span>wheel-flat</span>
        <Icon render="wheel-flat" />
      </Wrapper>
      <Wrapper>
        <span>windscreen-repair</span>
        <Icon render="windscreen-repair" />
      </Wrapper>
      <Wrapper>
        <span>windscreen-replacement</span>
        <Icon render="windscreen-replacement" />
      </Wrapper>
      <Wrapper>
        <span>windscreen</span>
        <Icon render="windscreen" />
      </Wrapper>
    </div>
  )
}

const Wrapper = styled.div`
  align-items: center;
  display: flex;
  justify-content: space-between;
  margin-bottom: 10px;
  width: 250px;
`
