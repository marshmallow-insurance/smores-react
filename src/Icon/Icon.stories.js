import React from 'react';
import {storiesOf} from '@storybook/react';

import {text} from '@storybook/addon-knobs';

import {Icon} from './Icon';

storiesOf('Icon', module).addWithJSX('default', () => (
  <Icon
    render={text('Render', 'contact-at')}
    size={text('Size (in px)', '32')}
    color={text('Color', 'blue7')}
  />
));
