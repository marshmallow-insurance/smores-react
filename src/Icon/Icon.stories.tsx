import React from 'react'
import { Icon, IconProps } from './Icon'
import { Container } from './Container'

export default {
  title: 'Icon',
  component: Icon,
}

const Template = (props: IconProps) => <Icon {...props} />

export const Default = Template.bind({})

Default.args = {
  render: 'info',
  size: 32,
  color: 'liquorice',
  rotate: '0',
  mt: '0',
  mr: '0',
  mb: '0',
  ml: '0',
}

const CollectionTemplate = () => <Container />

export const Collection = CollectionTemplate.bind({})

Collection.args = {
  size: 24,
  color: 'liquorice',
  rotate: '0',
  mt: '0',
  mr: '0',
  mb: '',
  ml: '0',
}
