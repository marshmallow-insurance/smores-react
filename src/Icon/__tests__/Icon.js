import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'

import { Icon } from '../Icon'

test('renders', () => {
  const { container } = render(<Icon render="at" size="32" color="pink" />)
  expect(container.firstChild).toMatchSnapshot()
})
