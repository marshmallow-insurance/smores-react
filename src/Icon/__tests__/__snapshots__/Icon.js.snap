// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  width: 32px;
  height: 32px;
  -webkit-transform: rotate(0deg);
  -ms-transform: rotate(0deg);
  transform: rotate(0deg);
}

<span
  class="c0"
  color="pink"
  rotate="0"
  size="32"
>
  <svg
    fill="none"
    height="100%"
    viewBox="0 0 24 24"
    width="100%"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M12 2C6.486 2 2 6.486 2 12s4.486 10 10 10a1 1 0 100-2c-4.411 0-8-3.589-8-8s3.589-8 8-8 8 3.589 8 8c0 1.103-.897 2-2 2s-2-.897-2-2c0-2.206-1.794-4-4-4s-4 1.794-4 4 1.794 4 4 4c1.2 0 2.266-.542 3-1.382.734.84 1.8 1.382 3 1.382 2.206 0 4-1.794 4-4 0-5.514-4.486-10-10-10zm0 12c-1.103 0-2-.897-2-2s.897-2 2-2 2 .897 2 2-.897 2-2 2z"
    />
  </svg>
</span>
`;
